src/wallet/rpcwallet.cpp

```diff
@@ -51,6 +51,19 @@ static inline bool GetAvoidReuseFlag(const CWallet* const pwallet, const UniValu
     return avoid_reuse;
 }
 
+static const RPCResult RESULT_LAST_PROCESSED_BLOCK { RPCResult::Type::OBJ, "lastprocessedblock", "hash and height of the block this information was generated on",{
+    {RPCResult::Type::STR_HEX, "hash", "hash of the block this information was generated on"},
+    {RPCResult::Type::NUM, "height", "height of the block this information was generated on"}}
+};
+
+void AppendLastProcessedBlock(UniValue& entry, const CWallet* const pwallet) EXCLUSIVE_LOCKS_REQUIRED(pwallet->cs_wallet)
```

Comment:
nit, to make sure this is not nullptr

```suggestion
void AppendLastProcessedBlock(UniValue& entry, const CWallet& wallet) EXCLUSIVE_LOCKS_REQUIRED(wallet.cs_wallet)
```

Reply:
Yeah, in *getbalances* they use a reference, but the other two functions use **pwallet**. My first thought was: let's change this to a references, but this would be an unrelated task. 

Who knows, maybe later, in a new PR: "transition from pwallet to wallet&"...if possible.